packages/react-query/src/scripts/utils/package.ts

```diff
@@ -49,3 +51,41 @@ export function getSuspensiveReactQueryPackageJson(targetVersion: string) {
 
   return packageJson
 }
+
+export function getIndexFileContent() {
+  const basePath = path.resolve(__dirname, '../../dist').replace(/src/, '')
+
+  return fs.readFileSync(path.join(basePath, 'index.js'), 'utf-8') || ''
+}
+
+export function getTargetSuspensiveReactQueryVersion(): string {
+  const indexFileContent = getIndexFileContent()
+
+  return (RegExp(/@suspensive\/react-query-(\d+)/).exec(indexFileContent) || [])[1] || ''
+}
+
+export function getTargetSuspensiveReactQueryAPIs() {
```

Comment:
Read the built JavaScript (index.js) file to check the current target @suspensive/react-query, parse it, and create a list of the exported APIs to display.